### PR TITLE
feat(python/sdk): allow input_text into lemur

### PR DESCRIPTION
**Pull request by jhazenaai**
Add the ability to call LeMUR with `input_text`. Update tests.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 setup(
     name="assemblyai",
-    version="0.19.0",
```
Comment (Swimburger) [inline setup.py:10]: Should this be `0.20.0`?

Comment (jhazenaai) [reply setup.py:10]: Good catch!

Comment (jhazenaai) [reply setup.py:10]: updated version number

**Review by Swimburger — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -266,6 +266,37 @@ print(result.response)
 
 </details>
 
+
+<details>
+  <summary>Use LeMUR to with Input Text</summary>
```
Comment (rodrahal) [inline README.md:271]: nit: Is there a typo here? "Use LeMUR to with"?

File: assemblyai/lemur.py [hunk 3]
```diff
@@ -121,7 +133,7 @@ class Lemur:
 
     def __init__(
         self,
-        sources: List[types.LemurSource],
+        sources: Optional[List[types.LemurSource]] = None,
```
Comment (rodrahal) [inline assemblyai/lemur.py:136]: q: Why is this changing? Was this always optional?

**Review by rodrahal — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jhazenaai) [reply README.md:271]: Ah yes. I think we should follow up on that in another change. The feedback loop is very long on this. Multiple PRs and 1+ hour of tests to run.

File: assemblyai/lemur.py (hunk 3 above)
Comment (jhazenaai) [reply assemblyai/lemur.py:136]: It's now optional. You can provide sources or input text.

**Pull request merged**